### PR TITLE
Rename the ClientWorld#netHandler field to networkHandler

### LINKED ISSUE #2912: `ClientWorld.netHandler` should be renamed to `networkHandler`
`networkHandler` is widely used in yarn.

### PR DESCRIPTION
**Pull request by haykam821**
Fixes #2912

### CODE REVIEW

**Review by Juuxel — APPROVED** [review]

**Review by apple502j — APPROVED** [review]

**Review by Shnupbups — APPROVED** [review]

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/client/world/ClientWorld.mapping [hunk 1]
```diff
@@ -14,7 +14,7 @@ CLASS net/minecraft/class_638 net/minecraft/client/world/ClientWorld
 	FIELD field_34804 chunkUpdaters Ljava/util/Deque;
 	FIELD field_35163 simulationDistance I
 	FIELD field_35432 BLOCK_MARKER_ITEMS Ljava/util/Set;
-	FIELD field_3727 netHandler Lnet/minecraft/class_634;
+	FIELD field_3727 networkHandler Lnet/minecraft/class_634;
```
Comment (enbrain) [inline mappings/net/minecraft/client/world/ClientWorld.mapping:17]: Can you fix the constructor parameter as well?

https://github.com/FabricMC/yarn/blob/29f3c7a07edca6483012bdc10c1a638519db7b57/mappings/net/minecraft/client/world/ClientWorld.mapping#L20-L21

Comment (liach) [reply mappings/net/minecraft/client/world/ClientWorld.mapping:17]: This line is right beyond the range for patch, so needs a quick edit (can be done on github ui)

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/client/world/ClientWorld.mapping (hunk 1 above)
Comment (modmuss50) [reply mappings/net/minecraft/client/world/ClientWorld.mapping:17]: @haykam821 are you able to quickly fix the arg^ Thanks 👍

**Comment by modmuss50** [comment]
Just going to merge this, please open another PR to fix the param.

**Pull request merged**